### PR TITLE
New layout & authentication

### PR DESCRIPTION
**Pull request by jotak**
Adopt a layout closer to patternfly standards with vertical navigation. It's also a bit closer to the alerts UI.
New section "Configuration" that gathers tenant and authentication configs.
New routes schema to allow easier server-side discrimination of angular routes

These changes needed some refactoring.

### CODE REVIEW

**Comment by jotak** [comment]
Some screenshots: http://imgur.com/SOFIGhh
http://imgur.com/gnAAvsK
http://imgur.com/j1xPVGN

File: metrics/src/app/app.module.ts [hunk 1]
```diff
@@ -42,10 +42,10 @@ import { ChartComponent } from './chart.component';
     HawkularChartsModule,
     RouterModule.forRoot([{
         path: '',
-        redirectTo: '/metrics',
+        redirectTo: '/r/metrics',
         pathMatch: 'full'
       }, {
-        path: 'metrics',
+        path: 'r/metrics',
```
Comment (mtho11) [inline metrics/src/app/app.module.ts]: What is the 'r' path addition for?

**Comment by jotak** [comment]
Fixes #11

**Comment by jotak** [comment]
Remark: the "Metrics" menu is not shown until a tenant is selected.
With hindsight I think it would be better to show it but disable it.

File: metrics/src/app/app.module.ts (hunk 1 above)
Comment (mtho11) [reply metrics/src/app/app.module.ts]: Oh, I see its for server side descrimination like the title says ;)

**Review by mtho11 — APPROVED** [review]
LGTM

**Pull request merged**